test/types/range/hilde/alignTypeError1.chpl

```diff
@@ -1,30 +1,50 @@
-writeln((1:int.. by 10 align 3:uint).type:string);
-writeln((1:int.. by 10 align 3:uint(64)).type:string);
-writeln((1:int(8).. by 10 align 3:uint).type:string);
-writeln((1:int(8).. by 10 align 3:uint(32)).type:string);
-writeln((1:int(8).. by 10 align 3:uint(64)).type:string);
-writeln((1:int(16).. by 10 align 3:uint).type:string);
-writeln((1:int(16).. by 10 align 3:uint(32)).type:string);
-writeln((1:int(16).. by 10 align 3:uint(64)).type:string);
-writeln((1:int(32).. by 10 align 3:uint).type:string);
-writeln((1:int(32).. by 10 align 3:uint(32)).type:string);
-writeln((1:int(32).. by 10 align 3:uint(64)).type:string);
-writeln((1:int(64).. by 10 align 3:uint).type:string);
-writeln((1:int(64).. by 10 align 3:uint(64)).type:string);
-writeln((1:uint.. by 10 align 3:int(8)).type:string);
-writeln((1:uint.. by 10 align 3:int(16)).type:string);
-writeln((1:uint.. by 10 align 3:int(32)).type:string);
-writeln((1:uint(8).. by 10 align 3:uint).type:string);
-writeln((1:uint(8).. by 10 align 3:uint(32)).type:string);
-writeln((1:uint(8).. by 10 align 3:uint(64)).type:string);
-writeln((1:uint(16).. by 10 align 3:uint).type:string);
-writeln((1:uint(16).. by 10 align 3:uint(32)).type:string);
-writeln((1:uint(16).. by 10 align 3:uint(64)).type:string);
-writeln((1:uint(32).. by 10 align 3:int(8)).type:string);
-writeln((1:uint(32).. by 10 align 3:int(16)).type:string);
-writeln((1:uint(32).. by 10 align 3:int(32)).type:string);
-writeln((1:uint(32).. by 10 align 3:uint).type:string);
-writeln((1:uint(32).. by 10 align 3:uint(64)).type:string);
-writeln((1:uint(64).. by 10 align 3:int(8)).type:string);
-writeln((1:uint(64).. by 10 align 3:int(16)).type:string);
-writeln((1:uint(64).. by 10 align 3:int(32)).type:string);
+var i8  = 3:int(8);
+var i16 = 3:int(16);
+var i32 = 3:int(32);
+var i64 = 3:int(64);
+var i   = 3:int;
+var u8  = 3:uint(8);
+var u16 = 3:uint(16);
+var u32 = 3:uint(32);
+var u64 = 3:uint(64);
+var u   = 3:uint;
+ 
+
```

Comment:
Looks like the line numbers changed in .good file. Why have these empty lines here?

I am ok keeping them if you have a good reason. It would be nice to explain
in a comment, to save other people from questioning.